DSharpPlus.Docs/DSharpPlus.docs.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I personally believe that this should be put in the `tools/` directory

Reply:
maybe